src/host/cache.rs

```diff
@@ -0,0 +1,13 @@
+use crate::host::kvpair::MerkleRecord;
+use lru::LruCache;
+use std::num::NonZeroUsize;
+use std::sync::Mutex;
+
+const DEFAULT_CACHE_SIZE: usize = 100000;
```

Comment:
Some docs said:
"If maxsize is set to None, the LRU feature is disabled and the cache can grow without bound. The LRU feature performs best when maxsize is a power-of-two."
so maybe change to  2**17?